distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex.enso

```diff
@@ -347,6 +349,10 @@ type Regex
                     Nothing -> Error.throw (No_Such_Group.Error name)
                     _ -> read_group_map groups name
                 case n of
+                    _ : Array ->
+                        # This case is reachable after https://github.com/oracle/graal/commit/59bbc050602d5045662bdd91e7b9be19b0839a3b
```

Comment:
Congrats on finding the [TRegex commit](https://github.com/oracle/graal/commit/59bbc050602d5045662bdd91e7b9be19b0839a3b) - great _software archeology_!